heroku_deploy.sh

```diff
@@ -12,7 +12,13 @@ do
  esac
 done
 
+
 if [ "${WEBUI:-false}" == "true" ]; then
+  STATUS_CODE=$(curl -s -o /dev/null -w '%{http_code}' -u ":$HEROKU_API_KEY" -n https://api.heroku.com/apps/${HEROKU_APP_NAME} -H "Accept: application/vnd.heroku+json; version=3");
+  if [ ${STATUS_CODE} -eq 403 ]; then
+    echo "You do not have access to this repository"
```

Comment:
Ah. makes sense. I hope Heroku fails gracefully if the app name is already taken. Considering this, wouldn't this be a problem with continuous integration from the webUI?